<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow::separator {
  width: 1px;
}

QToolButton {
	font-size: 13px;
}

QToolButton::menu-indicator{
  width: 0px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>0</number>
      </property>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="PlotWidget" name="plotWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>300</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPlainTextEdit" name="loggerWidget">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="tabStopWidth">
        <number>120</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuModules">
    <property name="title">
     <string>Mo&amp;dules</string>
    </property>
    <widget class="QMenu" name="menuModulesQuanty">
     <property name="title">
      <string>&amp;Quanty</string>
     </property>
     <addaction name="quantySetPath"/>
     <addaction name="separator"/>
     <addaction name="quantySaveInputAction"/>
     <addaction name="quantySaveAsInputAction"/>
     <addaction name="separator"/>
     <addaction name="quantyLoadCalculations"/>
     <addaction name="quantyRemoveAllCalculations"/>
     <addaction name="separator"/>
     <addaction name="quantyModuleHideAction"/>
    </widget>
    <addaction name="menuModulesQuanty"/>
   </widget>
   <addaction name="menuModules"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="quantySaveInputAction">
   <property name="text">
    <string>Save Input</string>
   </property>
   <property name="toolTip">
    <string>Save the Quanty Input File</string>
   </property>
  </action>
  <action name="quantySaveAsInputAction">
   <property name="text">
    <string>Save Input As...</string>
   </property>
  </action>
  <action name="quantyModuleShowAction">
   <property name="text">
    <string>Show Module</string>
   </property>
   <property name="toolTip">
    <string>Show the Quanty Module</string>
   </property>
  </action>
  <action name="quantyModuleHideAction">
   <property name="text">
    <string>Hide Module</string>
   </property>
   <property name="toolTip">
    <string>Hide the Quanty Module</string>
   </property>
  </action>
  <action name="quantyLoadCalculations">
   <property name="text">
    <string>Load Calculations</string>
   </property>
  </action>
  <action name="quantyRunCalculationAction">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="quantyRemoveAllCalculations">
   <property name="text">
    <string>Remove All Calculations</string>
   </property>
  </action>
  <action name="quantySetPath">
   <property name="text">
    <string>Set Path...</string>
   </property>
   <property name="toolTip">
    <string>Set Path to Quanty</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>.widgets.plotwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
